v2/java/src/main/resources/client_secrets.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
I like how the python client_secrets file is detailed, I would suggest keeping them consistent:
```{
  "installed": {
    "client_id": "[[INSERT CLIENT ID HERE]]",
    "client_secret": "[[INSERT CLIENT SECRET HERE]]",
    "redirect_uris": [],
    "auth_uri": "https://accounts.google.com/o/oauth2/auth",
    "token_uri": "https://accounts.google.com/o/oauth2/token"
  }
}
```